module-4/backend/week_four.md

```diff
@@ -4,16 +4,29 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What's your favorite project management tool?
+1. What's your favorite project management tool? 
++ I like Waffle for its simplicity. 
 2. Why do we create staging environments?
++ To ensure we can test our code before committing it to production. It is an oppotunity for test users to interact with the site in a safe testing environment. 
 3. What are the characteristics of a good README (in your opinion)?
++ One that outlines all the setup instructions in a detailed manner and one that has example use cases with code snippets. 
 4. What's one main improvement you're going to make to your code regarding accessibility issues?
++ Make it easier to use with a screen reader. 
 5. What are some basic security concerns to be aware of when building applications?
++ Always use HTTPS and to sanitize user inputs.
 6. Why is continuous integration helpful/important?
++ Continuous integration is useful for constantly pushing up new code to master. It's also useful in that, if you follow the tenants of continuous integration, it is very easy to roll back code because you have committed it so frequently. 
```

Comment:
it seems like you've described good git workflow. continuous integration tools run all tests before we merge something to master and it also tests our code in different ruby environments